src/Shared/E2ETesting/E2ETesting.targets

```diff
@@ -2,6 +2,23 @@
   <!-- Version of this SDK is set in global.json -->
   <Sdk Name="Yarn.MSBuild" />
 
+  <!-- Make sure the settings files get copied to the test output folder. -->
+  <ItemGroup>
+    <None Update="e2eTestSettings.ci.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="e2eTestSettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    
+    <Content Update="e2eTestSettings.ci.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Update="e2eTestSettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
Why do we have both `<None>` and `<Content>` entries for this - don't we only need one of them? And could the path be reduced to `e2eTestSettings*.json` to match both at once?

Reply:
I don't want this to fail if the item gets added as content, and AFAIK you can't mix two itemgroups in MSBuild. This is just safer. I've taken your suggestion of using a wildcard, though